cross-edits-analysis/author_analytics.py

```diff
@@ -0,0 +1,94 @@
+'''
+    Date: 6/11/2020
+    Author: Haoran Fei
+    Script to perform author-specific analytics, as outlined in part II of Preliminary Data
+    Analysis Planning.
+
+'''
+
+import sys
+import getopt
+#import pandas as pd
+import matplotlib.pyplot as plt
+import loader
+
+
+def main(argv): # pylint: disable=C0116
+
+    data_file_path = ""
+    range_start = 0
+    range_end = 0
+
+    try:
+        opts, _ = getopt.getopt(argv, "", ["path=", "start=", "stop="])
+    except getopt.GetoptError:
+        print("author_analytics.py --path <pattern_of_path_to_data_files> --start \
+            <start_of_file_index_range> --end <end_of_file_index_range>")
+        sys.exit(2)
+
+    for option, value in opts:
+        if option == "--path":
+            data_file_path = value
+        elif option == "--start":
+            range_start = int(value)
+        else:
+            range_end = int(value)
+
+    data_loader = loader.Loader()
+
+    if data_file_path[-5:] == ".json":
```

Comment:
Much of the code here can be merged with the article analytics, can you merge them in your next commit?

Reply:
Yes. I will refactor them.

Reply:
Absolutely.